### PR TITLE
FIX Explicitly disable browser cache on verification response

### PR DESCRIPTION
**Pull request by emteknetnz**
Fixes https://github.com/silverstripe/silverstripe-mfa/issues/414

May fix https://github.com/silverstripe/silverstripe-mfa/issues/393 (untested)

### CODE REVIEW

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Is there a practical way of writing a unit test for this?

**Comment by emteknetnz** [comment]
I've added a unit test for this

File: tests/php/Authenticator/LoginHandlerTest.php [hunk 1]
```diff
@@ -334,6 +335,23 @@ public function testStartVerificationIncludesACSRFToken()
...
+        $store = new SessionStore($member);
+        $handler->setStore($store);
+        $request = new HTTPRequest('GET', '/');
+        $request->setSession(new Session([]));
+        $request->setRouteParams(['Method' => 'basic-math']);
+        $middleware = HTTPCacheControlMiddleware::singleton();
+        $middleware->enableCache(true);
+        $this->assertSame('enabled', $middleware->getState());
+        $handler->startVerification($request);
+        $this->assertSame('disabled', $middleware->getState());
```
Comment (maxime-rainville) [inline tests/php/Authenticator/LoginHandlerTest.php:356]:
```suggestion
        $this->assertSame('disabled', $middleware->getState(), 'startVerification disables cache on HTTPCacheControlMiddleware so challenges never get cached');
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Looks good. I think we should explicitly say what the test is asserting otherwise the nekt person who looks at this will be very confused.

**Comment by emteknetnz** [comment]
@maxime-rainville I've added some comments explaining what we're testing

**Review by maxime-rainville — APPROVED** [review]
Looks good.

**Pull request merged**